components/browse-decks/DeckSearch.js

```diff
@@ -21,45 +20,81 @@ const SearchBox = styled.input`
   background: ${({ theme }) => theme.colors.white};
   padding: 10px 20px;
   border: 1px solid ${({ theme }) => theme.colors.border};
-  border-radius: 10px 0px 0px 10px;
+  border-radius: 10px;
   flex: 1;
   font-size: 18px;
+  :focus {
+    outline: none;
+    -webkit-box-shadow: 0px 0px 10px 2px ${({ theme }) => theme.colors.border};
+    -moz-box-shadow: 0px 0px 10px 2px ${({ theme }) => theme.colors.border};
```

Comment:
I am quite sure that styled-components autogenerates these flags. I don't think we should add them.